src/composables/draw/draw-select.composable.ts

```diff
@@ -54,7 +57,12 @@ export default function useDrawSelect() {
       pixel,
       feature => {
         const featureMatch = olArray.includes(drawnFeatures.value, feature)
-        if (featureMatch) {
+        if (
+          (featureMatch &&
+            (layersOpen.value || myMapsOpen.value) &&
+            drawnFeatures.value.length > 0) ||
+          (featureMatch && layers.value.length === 0)
```

Comment:
both conditions have `featureMatch`, right? can be simplified

Reply:
True, I'd prefer to keep as is though, since I still remember having had a hard time to get this working :-)